packages/playground/src/utils/helpers.ts

```diff
@@ -57,3 +57,15 @@ export function getDashboardURL(network: string) {
 export function getCardName(card: NodeGPUCardType): string {
   return card.vendor + " - " + card.device;
 }
+
+export function toGigaBytes(value?: string) {
+  const giga = 1024 ** 3;
+
+  if (!value) return 0;
+
+  const val = +value;
```

Comment:
I suggest accepting the value as a number directly instead of converting it from a string. What are your thoughts on this?